roles/hosted_engine_setup/tasks/search_available_network_subnet.yaml

```diff
@@ -29,24 +30,26 @@
 - name: Search for an available IPv6 subnet
   block:
     - name: Define 3rd chunk
-      set_fact:
+      ansible.builtin.set_fact:
         chunk: 1000
       when: chunk is not defined
     - name: Set 3rd chunk
-      set_fact:
+      ansible.builtin.set_fact:
         chunk: "{{ chunk|int + 45 }}"  # 200 tries
+      changed_when: true
```

Comment:
Needed here (and in some more `set_fact`)?
It's not `shell` or `command`...

Reply:
Thanks. Compared with current code, the next push:
- Removes all 'changed_when:' lines I added that were not reported by ansible-lint. Not sure why I added them originally, will ignore for now.
- Changes a few to 'false', when it makes sense.